pkg/metrics/metrics.go

```diff
@@ -234,12 +240,13 @@ func EmitSubMetric(sub *olmv1alpha1.Subscription) {
 	if sub.Spec == nil {
 		return
 	}
-	SubscriptionSyncCount.WithLabelValues(sub.GetName(), sub.Status.InstalledCSV, sub.Spec.Channel, sub.Spec.Package).Inc()
+	SubscriptionSyncCount.WithLabelValues(sub.GetName(), sub.Status.InstalledCSV, sub.Spec.Channel, sub.Spec.Package, installPlanApprovalToString(sub.Spec.InstallPlanApproval)).Inc()
```

Comment:
since `InstallPlanApproval` is just a type alias for `string`, it's fine to just call `string(sub.Spec.InstallPlanApproval)` everywhere.

Reply:
thanks, i made those changes!